felix/fv/workload/workload.go

```diff
@@ -438,9 +438,6 @@ func (w *Workload) LatencyTo(ip, port string) (time.Duration, string) {
 }
 
 func (w *Workload) SendPacketsTo(ip string, count int, size int) (error, string) {
-	if strings.Contains(ip, ":") {
```

Comment:
What's this change for?

Reply:
I'm not sure why this wasn't failing elsewhere when it was introduced (maybe not used for IPv6?), but `ping` fails when you pass an IPv6 address with brackets:
```$ ping [2001:db8:1::1]
ping: [2001:db8:1::1]: Name or service not known
$ ping 2001:db8:1::1
PING 2001:db8:1::1(2001:db8:1::1) 56 data bytes
64 bytes from 2001:db8:1::1: icmp_seq=1 ttl=64 time=0.030 ms
64 bytes from 2001:db8:1::1: icmp_seq=2 ttl=64 time=0.047 ms
64 bytes from 2001:db8:1::1: icmp_seq=3 ttl=64 time=0.042 ms
^[64 bytes from 2001:db8:1::1: icmp_seq=4 ttl=64 time=0.040 ms
^C
--- 2001:db8:1::1 ping statistics ---
4 packets transmitted, 4 received, 0% packet loss, time 3056ms
rtt min/avg/max/mdev = 0.030/0.039/0.047/0.006 ms
```
Since we don't pass port to `ping` anyways, I don't think we need the brackets (and pretty sure it's just plain wrong to have them here).